### PR TITLE
add rv3 to changelog

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

**Comment by cqc-melf** [comment]
the docs check is removed because of duplication in the build_and_test workflow

File: pytket/docs/changelog.rst [hunk 1]
```diff
@@ -33,6 +33,17 @@ API changes:
 * The ``QubitPauliOperator.from_OpenFermion`` and
   ``QubitPauliOperator.to_OpenFermion`` methods are removed.
 
+Major new features:
+
+* New methods for mapping logical to physical circuits for some ``Architecture``.
+  The new method will use a list of user given method, each of them suitable only 
+  for a specific set of sub circuits. Users can add there own methods if they want to.
+  Take a look on the `mapping_example <https://github.com/CQCL/pytket/blob/main/examples/mapping_example.ipynb>`_
+  notebook to find out more. All compiler passes in pytket are updated to use the new method.
```
Comment (cqc-alec) [inline pytket/docs/changelog.rst:42]: Let's leave out the reference to the notebook. (It isn't live yet,)

File: pytket/docs/changelog.rst [hunk 2]
```diff
@@ -33,6 +33,17 @@ API changes:
 * The ``QubitPauliOperator.from_OpenFermion`` and
   ``QubitPauliOperator.to_OpenFermion`` methods are removed.
 
+Major new features:
+
+* New methods for mapping logical to physical circuits for some ``Architecture``.
+  The new method will use a list of user given method, each of them suitable only
```
Comment (cqc-alec) [inline pytket/docs/changelog.rst:39]:
```suggestion
  The new method will use a list of user-given methods, each of them suitable only 
```

File: pytket/docs/changelog.rst [hunk 3]
```diff
@@ -33,6 +33,17 @@ API changes:
 * The ``QubitPauliOperator.from_OpenFermion`` and
   ``QubitPauliOperator.to_OpenFermion`` methods are removed.
 
+Major new features:
+
+* New methods for mapping logical to physical circuits for some ``Architecture``.
+  The new method will use a list of user given method, each of them suitable only 
+  for a specific set of sub circuits. Users can add there own methods if they want to.
```
Comment (cqc-alec) [inline pytket/docs/changelog.rst:40]:
```suggestion
  for a specific set of subcircuits. Users can add their own methods if they want to.
```

File: pytket/docs/changelog.rst (hunk 1 above)
Comment (cqc-alec) [inline pytket/docs/changelog.rst:42]:
```suggestion
  notebook to find out more. All compiler passes in pytket are updated to use the new methods.
```

File: pytket/docs/changelog.rst [hunk 4]
```diff
@@ -33,6 +33,17 @@ API changes:
...
+  Take a look on the `mapping_example <https://github.com/CQCL/pytket/blob/main/examples/mapping_example.ipynb>`_
+  notebook to find out more. All compiler passes in pytket are updated to use the new method.
+  The methods already given by pytket are ``LexiRouteRoutingMethod``,
+  ``LexiLabellingMethod``, ``MultiGateReorderRoutingMethod``,
+  ``BoxDecompositionRoutingMethod``, ``AASRouteRoutingMethod`` and ``AASLabellingMethod``.
```
Comment (cqc-alec) [inline pytket/docs/changelog.rst:45]: Leave out `BoxDecompositionMethod` (not merged yet).

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: pytket/docs/changelog.rst (hunk 1 above)
Comment (cqc-melf) [reply pytket/docs/changelog.rst]: I have removed it

File: pytket/docs/changelog.rst (hunk 4 above)
Comment (cqc-melf) [reply pytket/docs/changelog.rst]: I have removed it

Comment (cqc-melf) [reply pytket/docs/changelog.rst]: see [7114822](https://github.com/CQCL/tket/pull/263/commits/7114822c829f9819dd282c45b3910a86a297fc7d)

File: pytket/docs/changelog.rst (hunk 1 above)
Comment (cqc-melf) [reply pytket/docs/changelog.rst]: see [7114822](https://github.com/CQCL/tket/pull/263/commits/7114822c829f9819dd282c45b3910a86a297fc7d)

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**